Example/SettingsTableViewController.swift

```diff
@@ -107,11 +108,19 @@ class SettingsTableViewController: UITableViewController {
 
     func configure(field: UITextField?, for key: Config.Key) {
         field?.placeholder = Bundle.main.infoDictionary?[key.rawValue] as? String
-        field?.text = Config.userDefaults?.string(forKey: key.rawValue)
+        field?.text = getTextFieldValue(key: key, placeholderText: field?.placeholder)
+    }
+
+    func getTextFieldValue(key: Config.Key, placeholderText: String?) -> String? {
+        if predefinedKeys.contains(placeholderText ?? "") {
+            return Config.userDefaults?.string(forKey: key.rawValue) ?? ""
+        } else {
+            return placeholderText
+        }
```

Comment:
is `predefinedKeys` still useful ? What can be the case when `predefinedKeys.contains(placeholderText ?? "")` is `false` ?

Reply:
In the first launch scenario, we need to get the values from the plist and assign those text to the text field's. 
I can do this in [code](https://github.com/rakutentech/ios-miniapp/blob/master/Example/SettingsTableViewController.swift#L110) like below,
`field?.text = Config.userDefaults?.string(forKey: key.rawValue) ?? field?.placeholder` 

but if we do so, then the values of the textfields will be 'RAS_APPLICATION_IDENTIFIER' and 'RAS_SUBSCRIPTION_KEY'. Hence to avoid this scenario I added a small workaround

Reply:
in fact `field?.text = Config.userDefaults?.string(forKey: key.rawValue)` may be enough, but nvm, you did a good job, it's working well ;)